### PR TITLE
Add Perfview

### PR DESCRIPTION
**Pull request by NikiforovAll**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate <manifest>`, where `<manifest>` is the name of the manifest you're submitting?
- [ ] Have you tested your manifest locally with `winget install -m <manifest>`?

-----
I would like to add Perfview to winget to script my installations on windows. Currently, Perfview doesn't have an installer. 
Should I create an installer or is there a way to install binaries without one? Do you think is a good idea to add it to https://github.com/microsoft/perfview?

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/winget-pkgs/pull/4039)

### CODE REVIEW

File: manifests/Perfview/2.0.61.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+AppMoniker: Perfview
+Version: 2.0.61
+Publisher: Microsoft
+Author: Microsoft
+License: Copyright (c) .NET Foundation and Contributors
+LicenseUrl: https://github.com/microsoft/perfview/blob/master/LICENSE.TXT
+Homepage: https://github.com/microsoft/perfview/
+Description: PerfView is a free performance-analysis tool that helps isolate CPU and memory-related performance issues.
+Tags: "dotnet,dotnet-core,performance-analysis,performance,windows"
+Commands: "code"
```
Comment (NikiforovAll) [inline manifests/Perfview/2.0.61.yaml:12]: should be Perfview

**Comment by denelon** [comment]
We've updated the schema for manifests. The manifest needs to be updated before it will be able to pass validation.

**Comment by NikiforovAll** [comment]
Hi,
Sorry, but I'm not going to finish this one in near future, please consider closing it.

**Pull request closed without merging**